pkg/submitter/TxnSubmitter.go

```diff
@@ -1,11 +1,12 @@
 // Copyright (c) The Tellor Authors.
 // Licensed under the MIT License.
 
-package ops
+package submitter
```

Comment:
This file is small enough so lets put the function from  this file  in `submitter.go`